docs/book/src/reference/project-config.md

```diff
@@ -4,6 +4,8 @@
 
 The Project Config represents the configuration of a Kubebuilder project. All projects that are scaffolded with the CLI will generate the `PROJECT` file in the projects' root directory.
 
+The PROJECT file was introduced since Kubebuilder [release 3.0.0](https://github.com/kubernetes-sigs/kubebuilder/releases/tag/v3.0.0), where the Plugins design was implemented. When scaffolding an operator, the CLI will generate a PROJECT file in the project's root directory. It tracks all data used to perform scaffolds and stores information on resources and plugins in use. These data enable plugins to make more informed decisions during scaffolding, improving their usefulness ([see an example PROJECT file](https://github.com/kubernetes-sigs/kubebuilder/blob/6f1f8c43cfbf260c971037ff039cca86a0980006/testdata/project-v3-with-deploy-image/PROJECT#L2-L21)). 
+
 ## Versioning
```

Comment:
Could we ensure that we have here the info shared in the following email of this thread? https://groups.google.com/g/kubebuilder/c/5c4_Xc3Fjx0/m/TERhvqaPBQAJ

Also, could we use alias for the docs so that is easier for we keep it maintained?

The links should be at the bottom of the doc

I added some suggestions above ^

Reply:
Sure I've done the changes as you've recommended

Reply:
Hi @ashutosh887 it still not done, see the following suggestions: 
- (to make links using alias) https://github.com/kubernetes-sigs/kubebuilder/pull/3287/files#r1141269463
- (to add alias at the bottom) https://github.com/kubernetes-sigs/kubebuilder/pull/3287/files#r1141269536

Could you please ensure that all links are with alias and the alias are at the bottom?